### PR TITLE
FLUID-6153: Removing scrollbars from descriptions

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6153

### CODE REVIEW

File: src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl [hunk 1]
```diff
@@ -151,10 +151,11 @@ html {
                 .fl-prefsEditor-panel-descr {
                     clip: auto;
```
Comment (cindyli) [inline src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl]: According to [CSS clip property usage at w3schools](https://www.w3schools.com/cssref/pr_pos_clip.asp): `Note: The clip property does not work if "overflow:visible".` The use of `clip: auto` is unnecessary. 

Also wondering if the use of `overflow: visible` is needed. If not specified, the inherited `hidden` value seems good too.

Comment (jobara) [inline src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl]: I think we actually want the content to be visible though. So i figure it's better to ensure that it is or at least be explicit.

**Comment by jobara** [comment]
@cindyli ready for more review

**Comment by cindyli** [comment]
Merged at 56cc1caafba6907f1ecf93cc3baf17b0e357ee97

**Pull request merged**